src/index.js

```diff
@@ -54,7 +54,15 @@ commander
 		"Update lightning branch and create a PR to develop."
 	);
 
-commander.on("--help", () => help());
+commander.on("--help", () => {
+	help();
+
+	if (commander.verbose) {
+		const path = require("path").resolve(__dirname, "workflow.txt"); // eslint-disable-line
+		const tmp = utils.readFile(path);
```

Comment:
if you are not using the `tmp` variable, then combining these lines seems good..

`console.log(utils.readFile(path));`